tests/fe/mapping_q_manifold_01.cc

```diff
@@ -0,0 +1,193 @@
+#include <fstream>
```

Comment:
this test lacks the usual header. it also needs to `#include "../tests.h"` like all other tests.